spec/acceptance/plugins_spec.rb

```diff
@@ -11,6 +11,8 @@
       include pulpcore::plugin::deb
       include pulpcore::plugin::file
       include pulpcore::plugin::migration
+      include pulpcore::plugin::ostree
+      include pulpcore::plugin::python
```

Comment:
If these are both 3.15+ only you can use this. I chose <= 3.14 so grepping 3.14 on removal is a good strategy.
```suggestion
      unless versioncmp($facts['pulpcore_version'], '3.14') <= 0 {
        include pulpcore::plugin::ostree
        include pulpcore::plugin::python
      }
```